### DISCUSSION TITLE
Colab notebook crashing while trying to plot 3D interactive cloud using pyvista

### DISCUSSION THREAD

**Discussion by bijasn** [pyvista/pyvista#4696] [Q&A] [answered]
I was trying to plot an interactive 3D cloud using some randomly generated points with PyVista in Google Colab. But every time I tried to do this, my Colab notebook crashed. I tried changing the Jupyter notebook backends, including ipyvtklink, pythreejs, panel, and ipygany, but nothing worked. Here is the code:

```python
import pyvista as pv
import numpy as np

pv.set_jupyter_backend('ipyvtklink')
points = np.random.random((150, 3))
cloud = pv.wrap(points)
cloud.plot(render_points_as_spheres=True, point_size=20)
```
Copy
Can anyone help me with this?

**Comment by tkoyama010** [comment, marked as answer]
“Cloud ready” JavaScript plotting environments that make PyVista work seems not available on Google Colab.
